INSTALLING_PACKAGES.md

```diff
@@ -1,7 +1,26 @@
 # Repository installation
 
-To install only virtualization related packages, for example, you can
-install the open-power-host-os-virt metapackage:
+To install Host OS packages from a repository, you'll first need a
+system with a
+[supported GNU/Linux distribution](README.md#supported-gnulinux-distributions)
+and a yum configuration file pointing to an accessible yum repository
+containing the Host OS packages.
+
+If you've just executed the build scripts and want to use the local
+repository, your yum configuration file will be available at
+`result/repository_config/latest`. Otherwise, you'll need one pointing
+to the remote yum repository.
+
+It is a good idea to clean the yum cache to make sure you're getting the
+latest packages from the repositories:
+
+```
+sudo yum -c <path_to_yum_repository_config> clean all
+```
+
+Decide which packages or groups of packages you want to install. To
```

Comment:
See my comments in a previous commit about how to incremently offer knowledge to the reader: initially teach how to install a single ordinary package, then mention that groups of packages may be installed by installing metapackages and list them.